pm.mk

```diff
@@ -32,9 +32,12 @@ C88 := $(WINE) $(C88_DIR)/c88
 CC88 := $(WINE) $(C88_DIR)/cc88
 LC88 := $(WINE) $(C88_DIR)/lc88
 
-ifeq ($(MEM_MODEL), large)
-	LDFLAGS += -Ml
-	CFLAGS += -Ml
+LDFLAGS := -M
+CFLAGS := -M
+
+ifdef $(MEM_MODEL)
+	LDFLAGS += $(MEM_MODEL)
+	CFLAGS += $(MEM_MODEL)
```

Comment:
This will not work because += will always add a space between "-M" and $(MEM_MODEL)

Reply:
Yes, you're right. That shouldn't be very hard to do, I will when I have some free time.